vars/scyllaCloudLongevityPipeline.groovy

```diff
@@ -75,7 +75,7 @@ def call(Map pipelineParams) {
 
                             export SIRENADA_REGION=eu-west-1
                             export SIRENADA_NUMBER_OF_NODES=${params.n_db_nodes}
-                            export SIRENADA_CLUSTER_KEEP=true
+                            export SIRENADA_CLUSTER_KEEP=50
```

Comment:
can you add `wrap([$class: 'BuildUser'])` to this, so the correct user would be tagged ?
cause tight now it would always tag jenkins (cause there a fallback to the username in siren-tests)

Reply:
Added. Thanks